src/partials/home/demo-custom-chart.js

```diff
@@ -276,137 +274,130 @@ class MultipleAxes extends React.Component {
     const tickValues = this.getTickValues();
 
     return (
-      <div className="Benefits-demo fancyBorder">
-        <svg
-          style={styles.parent}
-          viewBox="0 0 450 350"
-          height={350}
-          width={450}
-        >
-          {/* Create stylistic elements */}
-          <rect x="0" y="0" width="10" height="30" fill="#f01616" />
-          <rect x="420" y="10" width="20" height="20" fill="#458ca8" />
+      <svg style={styles.parent} viewBox="0 0 450 350" height={350} width={450}>
+        {/* Create stylistic elements */}
+        <rect x="0" y="0" width="10" height="30" fill="#f01616" />
+        <rect x="420" y="10" width="20" height="20" fill="#458ca8" />
 
-          {/* Define labels */}
-          <VictoryLabel x={25} y={24} style={styles.title} text="An outlook" />
-          <VictoryLabel x={430} y={20} style={styles.labelNumber} text="1" />
-          <VictoryLabel
-            x={25}
-            y={55}
-            style={styles.labelOne}
-            text={"Economy \n % change on a year earlier"}
-          />
-          <VictoryLabel
-            x={425}
-            y={55}
-            style={styles.labelTwo}
-            text={"Dinosaur exports\n $bn"}
-          />
+        {/* Define labels */}
+        <VictoryLabel x={25} y={24} style={styles.title} text="An outlook" />
+        <VictoryLabel x={430} y={20} style={styles.labelNumber} text="1" />
+        <VictoryLabel
+          x={25}
+          y={55}
+          style={styles.labelOne}
+          text={"Economy \n % change on a year earlier"}
+        />
+        <VictoryLabel
```

Comment:
Just a general comment - i know this is the color contrast for 'dinosaur exports' is so very hard to read, as is the black text on dark red/brown.

Reply:
These should end up back on white